liblangutil/EVMVersion.cpp

```diff
@@ -19,8 +19,9 @@
  * EVM versioning.
  */
 
-#include <liblangutil/EVMVersion.h>
 #include <libevmasm/Instruction.h>
+#include <liblangutil/EVMVersion.h>
+
```

Comment:
Why?

Reply:
This is caused by clang-format. I'm not sure if it needs to be reverted. Do you need me to revert it?